sa/database.go

```diff
@@ -182,3 +183,10 @@ func initTables(dbMap *gorp.DbMap) {
 	dbMap.AddTableWithName(core.SignedCertificateTimestamp{}, "sctReceipts").SetKeys(true, "ID").SetVersionCol("LockCol")
 	dbMap.AddTableWithName(core.FQDNSet{}, "fqdnSets").SetKeys(true, "ID")
 }
+
+func safeSelectOne(m *gorp.DbMap, holder interface{}, q string, args ...interface{}) error {
+	if strings.ContainsAny(q, "'\"\n\r\\") {
+		return errors.New("Query contains unsafe character")
```

Comment:
It might be helpful to the caller if the unsafe character list were in the error message to save having to dig up this function.